<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myCalculator</class>
 <widget class="QWidget" name="myCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myCalculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black</string>
  </property>
  <widget class="QPushButton" name="btn_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 200 24pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 200 24pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 200 24pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_obelus">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 200 24pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>รท</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 200 24pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_times">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 200 24pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>ร</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 200 24pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 200 24pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 200 24pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 200 24pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_minus">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 200 24pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 200 24pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 200 24pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_plus">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 200 24pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_point">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 200 24pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_equal">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 200 24pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="display">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 87 24pt &quot;Montserrat&quot;;
</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Montserrat'; font-size:24pt; font-weight:80; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_del">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 200 24pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="display_result">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25,25,25);
color: white;
border-radius: 8px;
font: 87 15pt &quot;Montserrat&quot;;

</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
